web/viewer.js

```diff
@@ -1340,9 +1345,7 @@ function webViewerInitialized() {
 //document.getElementById('secondaryOpenFile').setAttribute('hidden', 'true');
 //#endif
 
-//#if !(FIREFOX || MOZCENTRAL)
-  var locale = PDFJS.locale || navigator.language;
-//#endif
+  var PDFJS = pdfjsLib.PDFJS;
```

Comment:
Same as above, it's just to avoid unneeded code churn. We can change that once we define where we will store options on the viewer side.